### PR TITLE
feat: add an ec2 instance

### PR DESCRIPTION
**Pull request by adamhjk**
Adding a fedora instance!

### CODE REVIEW

File: main.tf [hunk 1]
```diff
@@ -91,4 +91,12 @@ resource "aws_route_table_association" "public_subnet_asso" {
     count = length(var.public_subnet_cidrs)
     subnet_id = element(aws_subnet.public_subnets[*].id, count.index)
     route_table_id = aws_route_table.second_rt.id
+}
+
+resource "aws_instance" "fedora" {
+    ami = "ami-06b7f026f48cd3c6b"
+    instance_type = "t2.micro"
```
Comment (bobotclown42) [inline main.tf:103]: You need to add a key pair, or you won't be able to log in to your new server.

**Review by bobotclown42 — CHANGES_REQUESTED** [review]
You need to add a key pair, or you won't be able to log in.

File: main.tf (hunk 1 above)
Comment (adamhjk) [reply main.tf:103]: Good catch! I'll update it.

Comment (adamhjk) [reply main.tf:103]: Done!

**Comment by adamhjk** [comment]
Should be good now, @bobotclown42

File: main.tf (hunk 1 above)
Comment (adamhjk) [reply main.tf:103]: Done!

**Comment by adamhjk** [comment]
Should be gtg now, @bobotclown42

**Review by bobotclown42 — APPROVED** [review]
Looks good! I'll merge.

**Pull request merged**